src/SFA.DAS.Commitments.Application/Commands/UpdateApprenticeshipStatus/StopApprenticeshipCommandHandler.cs

```diff
@@ -100,7 +96,8 @@ private async Task SaveChange(StopApprenticeshipCommand command, Commitment comm
             historyService.TrackUpdate(apprenticeship, ApprenticeshipChangeType.ChangeOfStatus.ToString(), null, apprenticeship.Id, CallerType.Employer, command.UserId, apprenticeship.ProviderId, apprenticeship.EmployerAccountId, command.UserName);
             apprenticeship.PaymentStatus = PaymentStatus.Withdrawn;
             apprenticeship.StopDate = command.DateOfChange;
-           
+            apprenticeship.MadeRedundant = command.MadeRedundant;
+
             await _apprenticeshipRepository.StopApprenticeship(commitment.Id, command.ApprenticeshipId, command.DateOfChange);
```

Comment:
you will need to pass through the madeRedundant value to StopApprenticeship method and change sql code to store the value.

Reply:
Its working now, Thank you!